ci-bench-runner/Cargo.toml

```diff
@@ -12,7 +12,7 @@ hyper = { version = "0.14.27", default-features = false }
 jsonwebtoken = "9.1.0"
 # TODO: replace by a crates.io version once they publish it
 octocrab = { git = "https://github.com/XAMPPRocky/octocrab.git" }
-sentry = { version = "0.31.7", features = ["tracing"] }
+sentry = { version = "0.31.7", features = ["tracing", "ureq", "rustls"], default-features = false }
```

Comment:
Out of curiosity, what does this do?

Reply:
I added more of this information to the commit message.